.github/workflows/build-docker-image.yml

```diff
@@ -38,9 +38,72 @@ jobs:
       uses: docker/build-push-action@v2
       with:
         context: .
-        file: ./Dockerfile
+        file: ./linux/Dockerfile
+        # In case linux/s390x fails with tls issues:
+        # see https://github.com/testcontainers/moby-ryuk/pull/26
+        # and https://github.com/testcontainers/moby-ryuk/pull/40
         platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
         # Only push if we are publishing a release
         push: ${{ github.event_name == 'release' && github.event.action == 'published' }}
         # Use a 'temp' tag, that won't be pushed, for non-release builds
         tags: testcontainers/ryuk:${{ github.event.release.tag_name || 'temp' }}
+
+  build-windows:
```

Comment:
What about a matrix to share the build configuration?

Reply:
The Windows image requires other build steps, due to https://github.com/docker/build-push-action only working for Linux images. I think it's more maintainable to have a proper job for Windows instead of several `if` clauses in a combined Linux+Windows job.

Reply:
I need to check whether https://lippertmarkus.com/2021/11/30/win-multiarch-img-lin/ helps.